packages/realm-server/tests/auth-client-test.ts

```diff
@@ -7,93 +7,104 @@ import {
 } from '@cardstack/runtime-common/realm-auth-client';
 import { VirtualNetwork } from '@cardstack/runtime-common';
 import jwt from 'jsonwebtoken';
+import { basename } from 'path';
 
 function createJWT(expiresIn: string | number) {
   return jwt.sign({}, 'secret', { expiresIn });
 }
 
-module('realm-auth-client', function (assert) {
-  let client: RealmAuthClient;
+module(basename(__filename), function () {
```

Comment:
this is a tiny little boilerplate that we need to add to each test file made sure we can correlate filenames to test modules so we can filter them properly